docs/6-gridfs.md

```diff
@@ -0,0 +1,88 @@
+# MongoDB GridFS
```

Comment:
I'd rather see this doc after this one : https://github.com/thephpleague/flysystem-bundle/blob/3.x/docs/2-cloud-storage-providers.md instead of a new file

Reply:
With the 3 examples, this file would become very long and the navigation rather blurred. Technically it's not a "cloud storage provider", but a "database storage", it's not linked to a specific provider since the database can be hosted anywhere and the protocol used is implemented by other services like Azure Cosmos or Amazon DocumentDB (which are clouds).

That's similar to `3-interacting-with-ftp-and-sftp-servers.md`.

Reply:
Then it's okay 😄